### DISCUSSION TITLE
DuckDB wasm - syntax error

### DISCUSSION THREAD

**Discussion by kbatchu** [duckdb/duckdb-wasm#1829] [Q&A] [answered]
Hello,
DuckDB wasm throws an error when I try to execute this SQL. Error is - Uncaught Error Error: Parser Error: syntax error at or near "EPSG"
LINE 1: ...t_transform(st_geomfromwkb(geometry), 'EPSG:26918', 'EPSG:3857'), 3218.68)',st

`Create Or Replace Table tblFindQuery_3e720f51 As SELECT OBJECTID, ID, NAME, ALT_NAME, CROSS_ST, LONG_NAME, LABEL, BOROUGH, NGHBHD, ROUTES, TRANSFERS, COLOR, EXPRESS, CLOSED, geometry, ST_Buffer(ST_Transform(ST_GeomFromWKB(geometry),'EPSG:26918','EPSG:3857'), 3218.68) FROM read_parquet('http://localhost/WebGISApp/data/nyc_subway_stations.parquet')`

Can someone tell me what the error is?
Thanks,
Kiran

**Comment by szarnyasg** [comment]
Hi @kbatchu, did you load the spatial extension using `load spatial;`?

**Reply by kbatchu** [reply]
Yes. I made sure it was loaded.

**Reply by kbatchu** [reply, marked as answer]
I found the problem. I was trying to convert the WKB geometry again that was already converted using ST_GeomWKB().
